creator-node/test/fileManager.js

```diff
@@ -149,4 +163,113 @@ describe('test saveFileToIpfsFromFs', () => {
 
     assert.deepStrictEqual(originalSegmentBuf.compare(ipfsResp), 0)
   })
+
+  // ~~~~~~~~~~~~~~~~~~~~~~~~~ saveFileFromBuffer() TESTS ~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  /**
+   * Given: a file buffer is being saved to ipfs, fs, and db
+   * When: cnodeUserUUID is not present
+   * Then: an error is thrown
+   */
+  it('should throw error if cnodeUserUUID is not present', async () => {
+    const req = {
```

Comment:
this should be called reqOverride or something. overloading variable names is a risky practice